### ISSUE TITLE
cool!

### ISSUE THREAD

**Issue by aquynh** [SCUBSRGroup/Taint-Analyse#1]
this looks nice, but will you add a README in English?

keep it up, cheers!

**Comment by coolsmurfs** [comment]
Yes,sir,I will ,We will add a READ me in English and will add the symbolic execution to this project.Keep it up.Thanks sir.

**Comment by coolsmurfs** [comment]
We will add the test cases in the latter sir,wait for me!

**Comment by aquynh** [comment]
looking forward to that ;-)
